### PR TITLE
Revo Nozzle in-skirt container for the Voron 2.4 all Sizes

### PR DESCRIPTION
**Pull request by DatenThielt**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
A RevoNozzle in-skirt containe end container that is inserted into the side of a skirt.  Designed for the 2.4 250-350 Varients.

#### How was it tested? 
All parts have been printed and tested

#### Any background context you want to provide?
My hot ends were roaming around in a box and getting lost,  I saw the mod by @Andre#7463 (So I says) On discord for the trident and asked if I could remix it, But ended up being a full re-design as you cant put horizontal axels on a 2.4 skirt.

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ like so:
 || [Voron V0 SSR Mount](./DaGlo/Voron_V0_SSR_Mount/) | Voron V0 SSR Mount for the bottom | :heavy_check_mark: | :x: | :x: | :x: |
 |DaGrooch13 |[Tophat Exhaust Filter V0](./DaGrooch13/Tophat_Exhaust_Filter) |Activated charcoal exhaust filter with HEPA that replaces the rear tophat panel |:heavy_check_mark: |:x: |:x: |:x:|
 |Dan88 |[V0 one-part Skirts](./Dan88/V0skirt) |One part Skirts for the V0 |:heavy_check_mark: |:x: |:x: |:x:|
+|Daten |[Revo hotend skirt case](./Daten/Voron_2.4_Revo_Nozzle_Holder)|A skirt case for the Revo hotends to reside|:x:|:x:|:heavy_check_mark:|:x:|:x:|
```
Comment (cbc02009) [inline printer_mods/README.md:111]: this should go after DaStivi's mods

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by cbc02009** [comment]
TODO: check STLs

**Comment by DatenThielt** [comment]
Fixed, sorry about that!

**Comment by MapleLeafMakers** [comment]
There are 2 .step files included here, v8 looks like the correct one, v11 only has the skirt part. can v11 be deleted?

**Comment by DatenThielt** [comment]
Both were incorrect and I have now uploaded the matching .step to the STL files including the missing doors

File: printer_mods/Daten/Voron_2.4_Revo_Nozzle_Holder/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## Voron Revo skirt mod
+
+This is a modified skirt that lets you house the 3 current revo voron nozzles whilst they are not in use.  Currently only designed for the 350,  300 is possible, 250 is  small but I think I found a way to squeeze them all in
+
+Designed for the Boron 2.4 R1 and R2
```
Comment (MapleLeafMakers) [inline printer_mods/Daten/Voron_2.4_Revo_Nozzle_Holder/README.md:5]: typo?

**Comment by DatenThielt** [comment]
Moved the CAD folder to the correct place,  Renamed the step file and fixed the Typo sorry!

**Comment by MapleLeafMakers** [comment]
LGTM

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**